docs/dev/testservers.rst

```diff
@@ -9,7 +9,7 @@ Test Server 1-3
 Deployment via Bamboo. Only for branches of the ``ls1intum/Artemis`` repository, no forks.
 Guide available in the Artemis Developer Confluence Space: `Deploying changes to test server`_.
 
-.. _`Deploying changes to test server`: https://confluence.ase.in.tum.de/display/Artemis/Deploying+changes+to+test+server
+.. _`Deploying changes to test server`: https://confluence.ase.in.tum.de/display/ArTEMiS/GitHub+Deployment
```

Comment:
"Deployment via Bamboo" ->

```suggestion
.. _`Deploying changes to test server`: https://confluence.ase.in.tum.de/display/ArTEMiS/Bamboo+Deployment
```

Reply:
in the slack dev channel it said that github is default workflow to deploy sth to a test server. so I assumed we should like the github confluence page

Reply:
i changed the description before the link to github actions

Reply:
is deploying to 4,5, ... any different to 1-3?

Reply:
TS1 - TS6 and TS9 can all be deployed via GitHub labels.

A full (except TS9 😅) list of the servers and how they're deployed is available here: https://confluence.ase.in.tum.de/pages/viewpage.action?pageId=25252245 

Deployment guides:

https://confluence.ase.in.tum.de/display/ArTEMiS/GitHub+Deployment
https://confluence.ase.in.tum.de/display/ArTEMiS/Bamboo+Deployment

Reply:
@krusche is there any security reason why deployment guides are in confluence and not in the documentation?
Otherwise for the sake of open source development, it would be better to directly write it in the documentation

Reply:
Not that I know. Everything that does not include passwords or secrets should be moved to Github.
I skimmed over the two pages:

https://confluence.ase.in.tum.de/display/ArTEMiS/GitHub+Deployment
https://confluence.ase.in.tum.de/display/ArTEMiS/Bamboo+Deployment

and both should be moved to the Github documentation

https://confluence.ase.in.tum.de/pages/viewpage.action?pageId=25252245 does include passwords. Feel free to move the parts without passwords to Github as well, but double check that no secrets / passwords are leaked